test/util.ts

```diff
@@ -10,4 +14,20 @@ const url = new URL(import.meta.url)
 const dirname = fileURLToPath(url)
 const fixturesDir = resolve(dirname, '..', 'fixtures')
 
-export { fixturesDir, testLogger }
+const useFixture = async (fixtureName: string) => {
```

Comment:
I created this helper and refactored all tests in `bundler.test.ts` to use it because I realised our tests were not mimicking the structure observed in production. In production, the base path (e.g. `/project`) is a parent to both the source directory (`/project/netlify/edge-functions`) and the dist directory (`/project/.netlify/edge-functions-dist`), but in tests we were using as dist directory a temporary path that is something like `/var/tmp`.

With this helper, we copy the source directory into that temporary directory and make the base path.